crates/node_runtime/src/node_runtime.rs

```diff
@@ -110,13 +110,22 @@ pub trait NodeRuntime: Send + Sync {
 pub struct RealNodeRuntime {
     http: Arc<dyn HttpClient>,
     installation_lock: Mutex<()>,
+    #[cfg(target_os = "windows")]
+    system_root_environment: Option<String>,
 }
 
 impl RealNodeRuntime {
     pub fn new(http: Arc<dyn HttpClient>) -> Arc<dyn NodeRuntime> {
+        #[cfg(target_os = "windows")]
+        let system_root_environment = std::env::var("SYSTEMROOT")
+            .context("Missing environment variable: SYSTEMROOT!")
+            .log_err();
+
```

Comment:
In fact, there is no need to put it in the properties of Node Runtime, it makes more complex for this.

Command.creation_flags is a special for Windows, so just add before it.

Reply:
Since it's only used when running `npm install`, I agree with we dont have to store this value.